<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QFrame" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="3">
      <widget class="QLabel" name="borrowLabel">
       <property name="text">
        <string>List Peminjam, klik untuk memilih atau masukkan kode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="4" colspan="3">
      <widget class="QTableWidget" name="ItemSelect">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::AnyKeyPressed|QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed|QAbstractItemView::EditTrigger::SelectedClicked</set>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <property name="supportedDragActions">
        <set>Qt::DropAction::CopyAction|Qt::DropAction::MoveAction</set>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>32</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <row/>
       <column>
        <property name="text">
         <string>Sisa</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nama</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsEnabled</set>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string/>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QTableWidget" name="ItemSelect_2">
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row/>
       <column>
        <property name="text">
         <string>Nama Peminjam</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="SearchBar">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Cari Nama Peminjam</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pilih Item Yang Ingin Dipinjam</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <layout class="QGridLayout" name="gridLayout_2"/>
     </item>
     <item row="6" column="3" rowspan="2">
      <widget class="QPushButton" name="CodeBtn">
       <property name="text">
        <string>Masukkan Kode</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1" rowspan="2" colspan="2">
      <widget class="QPushButton" name="PijamBtn">
       <property name="text">
        <string>Pijam</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
